tests/regression/31-ikind-aware-ints/07-defexc-large.c

```diff
@@ -1,4 +1,4 @@
-//PARAM: --disable ana.int.interval --enable ana.int.def_exc --disable ana.int.enums
```

Comment:
Since this test wants to explicitly test def_exc regardless of whether we may change defaults in the future, I'd be better not to remove these options.

Reply:
I agree.